src/Integration/Binding/SolutionBindingOperation.Writer.cs

```diff
@@ -34,7 +22,7 @@ public SolutionRuleSetWriter(string sonarQubeProjectKey, IRuleSetGenerationFileS
         /// <param name="downloadedRuleSet"><see cref="RuleSet"/> that was downloaded from the server</param>
         /// <param name="fileNameSuffix">Rule set file suffix</param>
         /// <returns>Full file path of the file that was written out</returns>
-        public string WriteSolutionLevelRuleSet(string solutionFullPath, RuleSet downloadedRuleSet, string fileNameSuffix)
+        internal string PendWriteSolutionLevelRuleSet(string solutionFullPath, RuleSet downloadedRuleSet, string fileNameSuffix)
```

Comment:
The doc comment needs to be updated.

Naming: TFS refers to "pending changes", which are changes that have been written to disk but not yet committed to the source control system. "Pend" is being used in a difference sense here, and changes haven't yet been written to disk. Using another word (e.g. "Queue") would help avoid confusion.